x-pack/filebeat/module/salesforce/login/config/login.yml

```diff
@@ -0,0 +1,56 @@
+{{ if .httpjson.enabled }}
+
+type: httpjson
+
+interval: {{ .interval }}
+request.method: GET
+auth.oauth2:
+  enabled: true
+  client.id: {{ .clientid }}
+  client.secret: {{ .clientsecret}}
+  token_url: https://login.salesforce.com/services/oauth2/token
+  user: {{ .user }}
+  password: {{ .password }}
+request.url: {{ .url }}/services/data/v52.0/query

```

Comment:
Is it always `v52.0` or is there a chance that the end-user will have to modify it?

Reply:
The reason for keeping it as a constant would be to explicitly call out that we're supporting v52.0 as it's the stable version and we have tested it out. I don't think we should rely on the end-users to provide the correct version, the module might not work as expected if the user provides some incorrect value for that field. That's just my two cents. Let me know what you think of it. Thanks!

Reply:
Make sense now, thanks for clarifying :)